guides/common/modules/proc_installing-proxy-packages.adoc

```diff
@@ -1,10 +1,10 @@
-[id="configuring-foreman-proxy-repositories-el-{package-manager}"]
+[id="installing-foreman-proxy-packages-{package-manager}"]
```

Comment:
IMHO this file is either lacking a heading or should be considered as a snippet (> change the file name prefix).